src/main/java/com/revature/Flumblr/controllers/ExceptionController.java

```diff
@@ -40,4 +41,12 @@ public ResponseEntity<Map<String, Object>> handleInvalidTokenException(InvalidTo
         return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(map);
     }
 
+    @ExceptionHandler(ProfileNotFoundException.class)
```

Comment:
We need to discuss this. As I think we are trying to minimize the number of custom exceptions.